src/components/Forms/ContactFormFooter/ContactFormFooter.js

```diff
@@ -0,0 +1,97 @@
+import { useState } from 'react'
+import PropTypes from 'prop-types'
+import { useForm } from 'react-hook-form'
+import { FormAlert, onSubmitSuccess, onSubmitError } from '../'
+
+function ContactFormFooter() {
+  const [loading, setLoading] = useState(false)
+  const { register, handleSubmit, errors, reset } = useForm()
+
+  const onSubmit = async (formData, e) => {
+    e.preventDefault()
+    setLoading(true)
+
+    try {
+      const contactApiEndpoint = '/api/contact'
+      const options = {
+        method: 'POST',
+        body: JSON.stringify(formData),
+      }
+      const response = await fetch(contactApiEndpoint, options)
+      if (response.ok) {
+        onSubmitSuccess('Your form was successfully submitted.')
+        setLoading(false)
+        setPhone('')
+        reset()
+      }
+    } catch (error) {
+      onSubmitError('OOPS! Something went wrong, please try again later.')
+      setLoading(false)
+    }
+  }
+
+  return (
+    <form id="s2do-form" onSubmit={handleSubmit(onSubmit)}>
+      <div className="col-md-12">
+        <div className="form-group">
+          <label htmlFor="InputEmail" className="light-text">
+            Email
+            <sup>*</sup>
+          </label>
+          <input
+            type="email"
+            className="form-control"
+            name="email"
+            id="InputEmail"
+            placeholder="cam@example.com"
```

Comment:
Please use a similar/same placeholder that the other email inputs use. See example [here](https://github.com/Vets-Who-Code/vets-who-code-app/blob/ad219f60247ba5ae795d16c4e4847a48b23ee1af/src/components/Forms/ContactForm/ContactForm.js#L82)

Reply:
Was having issues for some reason with pushing changes from this branch back after updating. Created a new branch with changes and set the branch to merge into this one. Added @andrew-t-james as a reviewer.